### ISSUE TITLE
Sort generated required body and query/path parameters

### ISSUE THREAD

**Issue by oktavlachs** [koxudaxi/fastapi-code-generator#153]
Given OAS3 like this:
```
paths:
  /{ueId}/sdm-subscriptions:
    post:
      summary: subscribe to notifications
      operationId: Subscribe
      tags:
        - Subscription Creation
      parameters:
        - name: ueId
          in: path
          description: Identity of the user
          required: true
          schema:
            $ref: 'http://localhost:8081/29.571/16.7.0/TS29571_CommonData.yaml#/components/schemas/VarUeId'
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/SdmSubscription'
        required: true
```
Expected behaviour:
The required body is generated BEFORE the optional parameters in the Python code.

![image](https://user-images.githubusercontent.com/38793469/117134402-d46c2b80-ada5-11eb-9011-3af9da00d29e.png)

Actual behaviour:
The required body is generated AFTER the optional parameters in the Python code. I have to bring it into the right order by hand to solve the issue.

![image](https://user-images.githubusercontent.com/38793469/117134314-b7375d00-ada5-11eb-9f95-d104c17e5837.png)

Solution:
Put the body function parameter before the other function parameters at the beginning of the parameter list.

Absolutely minor bug, but I thought I report it. Thanks!

**Comment by koxudaxi** [comment]
@oktavlachs 
Thank you for creating this issue.
Oh, it's a bug.
I will fix the problem :cat:

**Comment by sa4zet** [comment]
I have the same problem, just with custom header
```python
        x_provider: str = Header(..., alias='x-provider'),
        body: List[AnyUrl],
```

**Comment by koxudaxi** [comment]
@oktavlachs @sa4zet 
I have released a fixed version as `0.2.7`

**Comment by sa4zet** [comment]
Yeah, the order is same, but you used ... to resolve the problem.
I solved this problem in `0.2.6` using this template:

```jinja
{% for arg in operation.get_argument_list(True) %}
  {% if arg.default == None %}{{arg}},{% endif %}
{% endfor %}
{% for arg in operation.get_argument_list(True) %}
  {% if arg.default != None %}{{arg}},{% endif %}
{% endfor %}
```

instead of this:
```jinja
{{operation.snake_case_arguments}}
```

Thanks for your work anyway!

**Comment by koxudaxi** [comment]
Yes. I didn't change the order.
I think the changing order may cause injecting a bug.
We usually expect the same order for each method.

**Comment by sa4zet** [comment]
> Yes. I didn't change the order.
> I think the changing order may cause injecting a bug.
> We usually expect the same order for each method.

It's not a problem, just an observation ^^

**Comment by oktavlachs** [comment]
@koxudaxi @sa4zet 
Can we close this issue?
